apps/data-pipeline/map-scraper/src/index.ts

```diff
@@ -0,0 +1,126 @@
+import { exit } from "node:process";
+import { database } from "@packages/db";
+import { mapLocation } from "@packages/db/schema";
+
+interface LocationListData {
+  catId: number;
+  lat: number;
+  lng: number;
+  id: number;
+  name: string;
+}
+
+interface LocationDetailData {
+  mediaUrlTypes: string[];
+  mediaUrls: string[];
+}
+
+interface BuildingLocation {
+  name: string;
+  lat: number;
+  lng: number;
+  imageURLs: string[];
+}
+
+const CATEGORIES: Set<number> = new Set([
+  8424, 8309, 8311, 8392, 8405, 44392, 44393, 44394, 44395, 44396, 44397, 44398, 44400, 44401,
+  44402, 44538, 44537, 44399, 8396, 11907, 8400, 10486, 11906, 11889, 8310, 8312, 8393, 8394, 8397,
+  8398, 8399, 8404, 8407, 8408, 11891, 11892, 11899, 11900, 11902, 21318, 8406, 11908, 11935,
+]);
+
+// The Concept3D API key is hardcoded here, which is generally a bad practice due to the security risks.
+// For truly sensitive keys, environment variables or a secrets manager would be used (as with DB_URL).
+// This approach was considered pragmatic for this specific utility and non-sensitive key.
+
+const LOCATIONS_LIST_API =
+  "https://api.concept3d.com/locations?map=463&key=0001085cc708b9cef47080f064612ca5";
+const LOCATIONS_DETAIL_API = (id: number) =>
```

Comment:
this is misleadingly named like a constant despite being a function; either name it in camelCase such that it is clearly a function or make it a true string constant